umbraco-workflow/installing-workflow.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+In this article, we will cover the steps required to install Umbraco Workflow on your website.
+
+## Prerequisites
+
+* [Umbraco Installation](../umbraco-cms/fundamentals/setup/install/)
```

Comment:
Would it make sense to specify it only works on Umbraco 11 - at least at the time of release?

Reply:
I'm not quite sure what the comment means 🤔The pre-reqs or the video?